src/pages/Bridge/index.tsx

```diff
@@ -142,6 +142,12 @@ export default function Bridge() {
   const isUnsupportedBridgeNetwork =
     networkOptionsPreset.find(network => network.chainId === chainId)?.tag === NetworkSwitcherTags.COMING_SOON
 
+  useEffect(() => {
+    if (activeTab === BridgeTab.BRIDGE) {
+      setTxsFilter(BridgeTxsFilter.RECENT)
+    }
+  })
```

Comment:
1. add dependency array. 
2. btw you can use `selectBridgeTransactions` selector instead of `selectBridgeFilteredTransactions` from `EcoBridge` and then you don't need to worried about `BridgeTxsFilter`